src/v1/Controllers/Cli/Reset.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * FusionSuite - Backend
+ * Copyright (C) 2022 FusionSuite
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace App\v1\Controllers\Cli;
+
+use Ahc\Cli\Input\Command;
+use Ahc\Cli\Output\Writer;
+use Illuminate\Database\Capsule\Manager as Capsule;
+use Illuminate\Events\Dispatcher;
+use Illuminate\Container\Container;
+
+class Reset extends Command
+{
+  public function __construct()
+  {
+    parent::__construct('reset', 'Reset the database');
+  }
+
+  public function execute()
+  {
+    $writer = new Writer();
+    $writer->comment('=> The database will be reset', true);
+
+    $config = include(__DIR__ . '/../../../config.php');
+
+    $capsule = new Capsule();
+    $capsule->addConnection($config['db']);
+    $capsule->setEventDispatcher(new Dispatcher(new Container()));
+    $capsule->setAsGlobal();
+    $capsule->bootEloquent();
+
+    $database_name = $config['db']['database'];
+
+    $writer->green(' -> Droping the database ');
+
+    try {
+      Capsule::statement("DROP DATABASE `{$database_name}`");
```

Comment:
If the user not have grant right on the database server, drop database and create it not not be possible.
I prefer list all tables and drop all; you solution can be OK only for devs

Reply:
I think this command is mainly (only?) intended for developers, that's why I chose this solution. I don't think an administrator wants to reset a database in production (it's at least not very common :)).

There's an issue when deleting tables: it has to be done in a specific order, or it could fail because of constraints on foreign keys. I'm aware there are no foreign keys yet, but I think we'll quickly need some.

While writing this answer, I checked how it is done in Laravel, and I think I'll found a solution here: https://github.com/laravel/framework/pull/29620/files

Reply:
Fixed, and it's way better, thanks :) (I forgot it was possible to disable the FK constraints!)